pkg/api/api.go

```diff
@@ -49,12 +49,12 @@ func summaryJobsByVariant(report, reportPrev sippyprocessingv1.TestReport) []sip
 			jobSummaryVariant = sippyv1.JobSummaryVariant{
 				Variant: v.VariantName,
 				PassRates: map[string]sippyv1.PassRate{
-					"latest": sippyv1.PassRate{
+					"latest": {
```

Comment:
imho this change reduces readability.  again not against it per se, but i am against it being enforced by the linter.

Reply:
This was from the linter expecting files to be ran through `gofmt` with the `-s` flag. I agree this particular change isn't great for readability, so I'll dial that one back.

Reply:
The linter will now just make sure the file is ran through `gofmt`, with no enforcement that the `-s` flag is used.

Reply:
sgtm